src/sentry/api/endpoints/organization_auth_provider_details.py

```diff
@@ -24,9 +20,6 @@ def get(self, request, organization):
         :pparam string organization_slug: the organization short name
         :auth: required
         """
-        if not features.has('organizations:sso-basic', organization, actor=request.user):
-            return Response(ERR_NO_SSO, status=status.HTTP_403_FORBIDDEN)
```

Comment:
Is the thinking here that if a customer can't add an SSO provider they will have nothing to get? Do customers get to keep their SSO providers if they downgrade?

Reply:
Actually I'm going to move this part to a separate PR, where it will make a bit more sense.